Resources/Prototypes/Entities/Structures/Machines/recycler.yml

```diff
@@ -33,8 +33,8 @@
       conveyor:
         shape: !type:PolygonShape
           vertices:
-          - -0.55,-0.55
-          - 0.55,-0.55
+          - -0.35,-0.55
+          - 0.35,-0.55
```

Comment:
Is there a reason the bounds are changed?

Reply:
It was to better fit how the recycler would realistically "grind". Gives leeway to bumping the sides and makes it feel more dimensional. (A grinder wouldn't grind something touching a flat outer wall used for safety/cosmetics/armor)